flex/views/analysis/AnalysisDownloadView.mxml

```diff
@@ -845,9 +845,8 @@
 				<mx:HierarchicalData source="{downloadList}"/>
 			</mx:dataProvider>						  
 			<mx:columns>
-				<mx:AdvancedDataGridColumn dataField="@displayName"  editable="false" headerText="Folder or File" width="165"/> 
-				<mx:AdvancedDataGridColumn dataField="@viewURL" headerText="" editable="false" width="20" itemRenderer="views.renderers.LinkButtonViewFile"/>
-				<mx:AdvancedDataGridColumn dataField="@fileSizeText" editable="false" headerText="Size" width="20" textAlign="right" /> 
+				<mx:AdvancedDataGridColumn dataField="@displayName"  editable="false" headerText="Folder or File" width="165" itemRenderer="views.renderers.AnalysisDownloadRenderer"/>
+				<mx:AdvancedDataGridColumn dataField="@fileSizeText" editable="false" headerText="Size" width="20" textAlign="right" />

```

Comment:
Ok, yes. Thanks I’ll make that change.

From: tmaness [mailto:notifications@github.com]
Sent: Wednesday, August 17, 2016 8:56 AM
To: hci-gnomex/gnomex <gnomex@noreply.github.com>
Cc: John Dewell <John.Dewell@hci.utah.edu>; Author <author@noreply.github.com>
Subject: Re: [hci-gnomex/gnomex] GNOMEX 2109 On the Analysis screen, files tab and a viewable is shown… (#10)


In flex/views/analysis/AnalysisDownloadView.mxml<https://github.com/hci-gnomex/gnomex/pull/10#discussion_r75139127>:

> @@ -845,9 +845,8 @@

>                              <mx:HierarchicalData source="{downloadList}"/>

>                      </mx:dataProvider>

>                      <mx:columns>

> -                            <mx:AdvancedDataGridColumn dataField="@displayName"  editable="false" headerText="Folder or File" width="165"/>

> -                            <mx:AdvancedDataGridColumn dataField="@viewURL" headerText="" editable="false" width="20" itemRenderer="views.renderers.LinkButtonViewFile"/>

> -                            <mx:AdvancedDataGridColumn dataField="@fileSizeText" editable="false" headerText="Size" width="20" textAlign="right" />

> +                            <mx:AdvancedDataGridColumn dataField="@displayName"  editable="false" headerText="Folder or File" width="165" itemRenderer="views.renderers.AnalysisDownloadRenderer"/>

> +                            <mx:AdvancedDataGridColumn dataField="@fileSizeText" editable="false" headerText="Size" width="20" textAlign="right" />

You might want to make the width for @displayName<https://github.com/displayName> 185 now @viewURL is gone...

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/hci-gnomex/gnomex/pull/10/files/b2b396019dec37a1c48ab8a5ba33822ab816c545#r75139127>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AI14N_Ae2woXLdz_qQCPoS788hHBKZKYks5qgyDsgaJpZM4JmhjB>.